diff --git a/node_modules/@rehooks/component-size/index.js b/node_modules/@rehooks/component-size/index.js
index 5e652a2..c30c287 100644
--- a/node_modules/@rehooks/component-size/index.js
+++ b/node_modules/@rehooks/component-size/index.js
@@ -31,6 +31,9 @@ function useComponentSize(ref) {
     },
     [ref]
   )
+  if (typeof window === 'undefined') {
+    return ComponentSize;
+  }
 
   useLayoutEffect(
     function() {
