diff --git a/node_modules/react-toastify/dist/core/toast.d.ts b/node_modules/react-toastify/dist/core/toast.d.ts
index 319accc..73021d3 100644
--- a/node_modules/react-toastify/dist/core/toast.d.ts
+++ b/node_modules/react-toastify/dist/core/toast.d.ts
@@ -53,7 +53,7 @@ declare const toast: {
         DEFAULT: TypeOptions;
     };
 };
-interface ToastPromiseParams {
+export interface ToastPromiseParams {
     pending?: string | UpdateOptions;
     success?: string | UpdateOptions;
     error?: string | UpdateOptions;
